cmk/gui/plugins/wato/check_parameters/netapp_luns.py

```diff
@@ -57,6 +58,32 @@ def _parameter_valuespec_netapp_luns():
                      )
                  ],
              )),
+            ("magic",
```

Comment:
This duplicates code. At least we should make `_filesystem_magic_elements` in `cmk/gui/plugins/wato/check_parameters/filesystem_utils.py` "public" (strip the leading underscore), and use it here. I think we could consolidate even more, but at least we should not add more duplicate code....

Reply:
Hi Moritz - nightshift today? :)
inside the filesystem_utils.py the part size_trend_elements is already public. Then the same for magic_elements would be easiest way i think.